### ISSUE TITLE
Multiple Dataflows don't update

### ISSUE THREAD

**Issue by chenerlich** [yeti-platform/yeti#287]
### Description
I integrated the platform about a month ago, and now all the updated data shown in "Browse" is 99% phising.
When i look into the dataflows table, i see that while there are feeds that get updated, there are also feeds that don't.
I restarted the system\workers\schedulers a few times, and pulled and "syncdb" the repository to its latest state.
No change regarding those dataflows.
Attached a screenshot for few for few of them:

![capture](https://user-images.githubusercontent.com/17673075/44708510-020a5e00-aab0-11e8-809a-073e9eb6a422.JPG) (Today is the 28/08/2018...)

**Comment by tomchop** [comment]
Yeah, this seems to be about Mongo & celery not getting along well and deadlocking at some point, especially in smaller servers. What kind of specs are you running Yeti on? 

For the time being, you can "unlock" feeds using this mongo script:

```
use yeti;
db.schedule_entry.update({lock: true}, {$set: {lock: false, status: "Unlocked..."}}, {multi: true});
```

**Comment by tomchop** [comment]
I guess more CPUs / cores couldn't hurt. Can you show me how the services are started? The default is to start 8 processes for feeds and 10 for analytics, so you have 18+ processes hitting mongodb simultaneously on a 4 CPU machine.

**Comment by tomchop** [comment]
You could also try decreasing the amount of workers (check the systemd scripts) in each service to something more adapted to your instance.

**Comment by chenerlich** [comment]
@tomchop 
Changed specs to 8 CPUs, 32 GB RAM.
It looks like this:
![capture](https://user-images.githubusercontent.com/17673075/44772523-c5556a00-ab76-11e8-8dd2-3aeb30b61d7a.JPG)

![capture2](https://user-images.githubusercontent.com/17673075/44772596-f59d0880-ab76-11e8-9eff-512982195789.JPG)

**Comment by tomchop** [comment]
TLDR: Yes, I would: a) stop all feeds, b) run script, c) relaunch feeds

The longer answer is that we store the state of the feeds in the db so that the same feed is not launched a second time while it's already running. If they deadlock, the feed is still marked as "running" in the database, but not actually running. Worst case, it keeps a whole worker busy, best case it will never run before it is unlocked. This is a bit hard to reproduce, but after the time we've spent on this we're pretty sure it comes from Celery and Mongo not working together nicely.

**Comment by chenerlich** [comment]
Worked great.
Thanks!

**Comment by sebdraven** [comment]
Perfect ! I'ce closed this issue !

**Comment by jalbrizio** [comment]
thanks for this, I ran into the same issue
